src/Search/SingleHackerView.tsx

```diff
@@ -0,0 +1,208 @@
+        <H1 marginLeft="0">{`${account.firstName} ${account.lastName}`}</H1>
+        <Form>
+          <Flex
+            width="100%"
+            flexWrap="wrap"
+            justifyContent="center"
+            alignItems="center"
+          >
+            <Box width={[1, 1 / 2]}>
+              <StyledSelect
+                isTight={true}
+                className="react-select-container"
+                classNamePrefix="react-select"
+                options={getOptionsFromEnum(HackerStatus)}
+                isDisabled={!this.state.canEdit}
+                onChange={this.handleChange}
+                value={{
+                  label: this.state.status,
+                  value: this.state.status,
+                }}
+              />
+            </Box>
+            <Flex width={[1, 1 / 2]} justifyContent={['center', 'flex-start']}>
+              <Button
+                type="button"
+                onClick={this.submit}
+                isLoading={this.state.isLoading}
+                disabled={this.state.isLoading}
+              >
+                Change status
+              </Button>
+            </Flex>
+          </Flex>
+        </Form>
+        <hr />
+        <Box ml="6px">
+          <H2 color={theme.colors.grey}>Basic Information</H2>
+          <Flex
+            width="100%"
+            flexWrap="wrap"
+            justifyContent="space-between"
+            alignItems="center"
+          >
+            <Box width={[1, 1 / 2]}>
+              <strong>Email</strong>: {account.email}
+            </Box>
+            <Box width={[1, 1 / 2]}>
+              <strong>School</strong>: {hacker.school}
+            </Box>
+            <Box width={[1, 1 / 2]}>
+              <strong>Degree</strong>: {hacker.degree}
+            </Box>
+            <Box width={[1, 1 / 2]}>
+              <strong>Status</strong>: {hacker.status}
+            </Box>
+            <Box width={[1, 1 / 2]}>
+              <strong>Graduation Year</strong>: {hacker.graduationYear}
+            </Box>
+            <Box width={[1, 1 / 2]}>
+              <strong>Major(s)</strong>: {hacker.major}
+            </Box>
+            <Box width={[1, 1 / 2]}>
+              <strong>Skills</strong>:{' '}
+              {hacker.application.skills
+                ? hacker.application.skills.join(', ')
+                : 'None'}
+            </Box>
+            <Box width={[1, 1 / 2]}>
+              <strong>Job interest</strong>: {hacker.application.jobInterest}
+            </Box>
+          </Flex>
+          <hr />
+          <H2 color={theme.colors.grey}>Links</H2>
+          <Flex
+            width="100%"
+            flexWrap="wrap"
+            justifyContent="space-between"
+            alignItems="center"
+          >
+            <Box width={[1, 1 / 2]}>
+              <strong>GitHub</strong>:{' '}
+              {hacker.application.portfolioURL.github ? (
+                <a href={hacker.application.portfolioURL.github}>
```

Comment:
Can we add `target="_blank"` so that these links open up into a new tab?

Reply:
Could we refactor this into a `LinkView` component?

Reply:
What's a `LinkView`?